web_src/css/base.css

```diff
@@ -983,8 +975,17 @@ img.ui.avatar,
   margin-right: 0 !important;
 }
 
-@media (max-width: 767px) {
-  #navbar:not(.shown) > *:not(:first-child) {
+@media (max-width: 767.98px) {
+  #navbar {
+    align-items: stretch;
+  }
+  #navbar.shown {
```

Comment:
Where is this `shown` used? Did a search but didn't find it, is it from fomantic?

Reply:
It is our own navbar toggle feature, only visible on mobile widths:

https://github.com/go-gitea/gitea/blob/0d47b0ab1de6ccbbead3cc7eae185fb3c50c47fe/web_src/js/features/common-global.js#L22-L36

Reply:
I guess it can be refactored to `showElem` and `hideElem`.

Reply:
Actually no, this can not be refactored to use those because the default state is dependant on the viewport width, so I can not add a static class to the elements without also affecting the desktop viewports.

Reply:
I renamed `shown` class to `navbar-menu-open` for clarity and refactored the toggle mechanism to vanilla js in https://github.com/go-gitea/gitea/pull/25134/commits/ba4ecf10423e8b2660960d53fb879f7168b6d790.